src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-item">
+          Surname:
+          <input type="text" name="Surname">
+        </label>
+
+        <label class="form-item">
+          Name:
+          <input type="text" name="Name">
+        </label>
+
+        <label class="form-item">
+          How old are You?
+          <input type="number"
+          name="Age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+
+        <label class="form-item">
+          Full date of birth:
+          <input type="date" name="BirthDate">
+        </label>
+
+        <label class="form-item">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" name="Agreement">
+          </label>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-item">
+          E-mail:
+          <input type="email"
+          name="Email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+
+        <label class="form-item">
+          Password:
+          <input type="password"
+          name="Password"
+          minlength="6"
+          maxlength="20"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-item">
```

Comment:
It should not be a label, just use div

fix all such cases

Reply:
[FUNCTIONAL] - You need to use a label tag for each input, so that every input could be activated by clicking on the corresponding label. 
This is 7th point of [CHECKLIST].